Sources/ComplexModule/Complex+Codable.swift

```diff
@@ -12,6 +12,7 @@
 import RealModule
 
 // FloatingPoint does not refine Codable, so this is a conditional conformance.
+@_unavailableInEmbedded
```

Comment:
I believe that these will need to be wrapped as follows:

```suggestion
#if compiler(>=6.0)
@_unavailableInEmbedded
#endif
```